src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -0,0 +1,590 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics.CodeAnalysis;
+
+using JSObject = Interop.JavaScript.JSObject;
+using JSException = Interop.JavaScript.JSException;
+using HostObject = Interop.JavaScript.HostObject;
+using Uint8Array = Interop.JavaScript.Uint8Array;
+using Function = Interop.JavaScript.Function;
+
+namespace System.Net.Http
+{
+    internal partial class BrowserHttpHandler : HttpMessageHandler
+    {
+        // This partial implementation contains members common to Browser WebAssembly running on .NET Core.
+        private static readonly JSObject? fetch = (JSObject)Interop.Runtime.GetGlobalObject("fetch");
+        private static readonly JSObject? window = (JSObject)Interop.Runtime.GetGlobalObject("window");
+
+        /// <summary>
+        /// Gets whether the current Browser supports streaming responses
+        /// </summary>
+        private static bool StreamingSupported { get; }
+
+        static BrowserHttpHandler()
+        {
+            using (var streamingSupported = new Function("return typeof Response !== 'undefined' && 'body' in Response.prototype && typeof ReadableStream === 'function'"))
+                StreamingSupported = (bool)streamingSupported.Call();
+        }
+
+        public bool UseCookies
+        {
+            get => throw new PlatformNotSupportedException("Property UseCookies is not supported.");
+            set => throw new PlatformNotSupportedException("Property UseCookies is not supported.");
+        }
+
+        [AllowNull]
```

Comment:
Is this needed?

Reply:
It will build without it but the original code had it so followed what was there.

I will take it out.

Reply:
Addressed in recent commit